packages/core/src/codegen/service.ts

```diff
@@ -87,25 +86,4 @@ export class CodegenService extends Emittery {
       msg: `Wrote new file at generated/index.ts`,
     });
   }
-
-  generateSchemaFile({ graphqlSchema }: { graphqlSchema: GraphQLSchema }) {
```

Comment:
Per results of the poll, let's add this back in. We can rename it `generateGraphqlSchemaFile` to make it more clear.

Also with that in mind, perhaps it does make more sense to call `buildGraphqlSchema` within the BuildService and emit both `schema` and `graphqlSchema` in the `newSchema` event.